packages/strapi-admin/admin/src/app.js

```diff
@@ -57,13 +57,23 @@ import history from './utils/history';
 
 import plugins from './plugins';
 
+import itly from './itly';
+import TelemetryPlugin from './telemetry-plugin';
+
 const strapi = Strapi();
 
 const initialState = {};
 const store = configureStore(initialState, history);
 const { dispatch } = store;
 const MOUNT_NODE = document.getElementById('app') || document.createElement('div');
 
+itly.load({
```

Comment:
Need to set `environment` to an appropriate value (either `development` or `production`). Defaults to `development`.